lib/travis/build/git/clone.rb

```diff
@@ -56,6 +72,14 @@ def quiet?
             config[:git][:quiet]
           end
 
+          def sparseCheckout
+            config[:git][:sparseCheckout]
+          end
+
+          def sparseCheckout?
+            !!config[:git][:sparseCheckout]
```

Comment:
oops, the correct one looks clear now, but I think it should be: if `sparseCheckout` is set, it'll return true, right? We only enable sparseCheckout if the value is set, if empty, we don't use it.